### DISCUSSION TITLE
Is there another LTS after 20.10.x?

### DISCUSSION THREAD

**Discussion by Marshalzxy** [moby/moby#47598] [Q&A] [answered]
After 2023.12, [Mirantis annouced 20.10 was end of life](https://docs.mirantis.com/mcr/23.0/compat-matrix/maintenance-lifecycle.html#maintenance-lifecycle). At this moment(2024.04),there are three Moby main releases: 23.x,24,x,25.x. I need to choose an main version that is stable and  will be supported by Moby for 2-1.5 years at least. Could anyone give me some suggestion about which main release should I choose?

**Comment by cpuguy83** [comment, marked as answer]
Mirantis sells long term support.

I also think we do a pretty good job at back porting bug fixes to prior releases.

Also, generally speaking, if you pin your API version, new versions won't have behavior changes.

**Comment by neersighted** [comment]
To clarify a bit further, Mirantis and Microsoft are collaborating to maintain the 23.0 branch. You won't get Docker CE (i.e. https://download.docker.com) packages, but Mirantis is happy to sell them to you, and you of course have access to the source if you want to roll up your sleeves.

You might also be interested in https://github.com/moby/moby/discussions/47328 and https://github.com/moby/moby/pull/46772.
